features/id-check-wrapper/internal/src/test/kotlin/uk/gov/onelogin/criorchestrator/features/idcheckwrapper/internal/screen/SyncIdCheckViewModelTest.kt

```diff
@@ -480,7 +497,7 @@ class SyncIdCheckViewModelTest {
             viewModel.onScreenStart(documentVariety = documentVariety)
             advanceUntilIdle()
 
-            assertEquals(1, configStore.getReadSingleCount())
```

Comment:
Is this change expected or might it indicate a bug?

What are the extra 3 calls to `readSingle()` for?

Reply:
I've had a go at improving this test here #570